app/jest.config.js

```diff
@@ -0,0 +1,28 @@
+// jest.config.js
+module.exports = {
+  verbose: true,
+  roots: [
+    "./client"
+  ],
+  transform: {
+    "^.+\\.[t|j]sx?$": "babel-jest"
```

Comment:
we should use "ts-jest" here to transform tsx elements. we might need more coverage on the regex for ts/js elements too but either way ts-jest will allow us to test on tsx files

we can either do it here or i can add it in after the fact when i start writing tests in the next PR!

Reply:
good point - updated https://kulshekhar.github.io/ts-jest/user/config/#jest-config-with-helper

> `ts-jest/presets/js-with-babel` | TypeScript files will be handled by ts-jest, and JavaScript files will be handled by babel-jest.